easybuild/easyblocks/i/icc.py

```diff
@@ -196,18 +193,26 @@ def make_module_req_guess(self):
             # 'lib/intel64' is deliberately listed last, so it gets precedence over subdirs
             guesses['LD_LIBRARY_PATH'].append('lib/intel64')
 
-        guesses['LIBRARY_PATH'] = guesses['LD_LIBRARY_PATH']
```

Comment:
I'm included to keep this, for consistency with other compilers... It doesn't hurt to have this set, right?